install.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Checking dependencies
+if ! [ -x "$(command -v firefox)" ]; then
+    echo 'Error: Firefox is not installed.' >&2
+    exit 1
+fi
+
+if ! [ -x "$(command -v python3)" ] && [ -x "$(command -v python)" ]; then
+    echo 'Error: Python is not installed.' >&2
+    exit 1
+fi
+
+PYTHON_VERSION=$(python3 --version || python --version)
+PYTHON_MAJOR_VERSION=$(echo $PYTHON_VERSION | awk '{print $2}'| awk -F'.' '{print $1}')
+
+if [[ $PYTHON_MAJOR_VERSION -ne 3 ]]; then
+    echo 'Python3 is in a version different than 3' >&2
+    exit 1
+fi
+
+# Check virtualenv instalation and run setup for application
+# set +u because virtualenv fails without option
+if ! [ -x "$(virtualenv --version)" ]; then
+    if ! [ "$(python -m venv -h)" ]; then
```

Comment:
The scipt install.sh was not working in my computer. 
```shell
/usr/bin/python: No module named venv
Virtualenv is not installed
```

It worked after a little change in line 27
```shell
if ! [ "$(python3 -m venv -h)" ]; then
```